.github/CONTRIBUTING.md

```diff
@@ -9,66 +9,56 @@ git clone https://github.com/transloadit/uppy.git
 cd uppy
 ```
 
-We are using [Corepack][] to manage version of [Yarn][]. Corepack comes pre-installed with Node.js >=16.x, or can be installed through `npm`. You can run `corepack enable` to install a `yarn` executable in your `$PATH`, or prefix all yarn commands with `corepack yarn`.
+We are using [Corepack][] to manage versions of [Yarn][].\
```

Comment:
```suggestion
We are using [Corepack][] to manage versions of [Yarn][].
```

Reply:
I actually enjoyed these "Corepack comes pre-installed with" comments (wasn't familiar with corepack previously)

Reply:
It's still there, on the very next line :) My suggestion is to remove the `\` char.

Reply:
Ah I see, because github renders newlines now even without double spacing or `\`?